embedded-bins/runc/Dockerfile

```diff
@@ -2,31 +2,36 @@ ARG BUILDIMAGE
 FROM $BUILDIMAGE AS build
 
 RUN apk add build-base git \
-	curl linux-headers gperf bash pkgconf
+	curl linux-headers gperf bash pkgconf patch cmd:tail
 
 ENV GOPATH=/go
 
 ARG LIBSECCOMP_VERSION=2.5.5
 RUN curl --proto '=https' --tlsv1.2 -L https://github.com/seccomp/libseccomp/releases/download/v$LIBSECCOMP_VERSION/libseccomp-$LIBSECCOMP_VERSION.tar.gz \
 	| tar -C / -zx
 
-RUN cd /libseccomp-$LIBSECCOMP_VERSION && ./configure --sysconfdir=/etc --enable-static
+WORKDIR /libseccomp-$LIBSECCOMP_VERSION
+# backport commit that lets us run the tests in parallel
+RUN curl -L https://github.com/seccomp/libseccomp/commit/2380f5788c692796f75e464c61aa877e5c4eb882.patch \
+	| patch -p1
+RUN ./configure --sysconfdir=/etc --enable-static
 
-RUN make -j$(nproc) -C /libseccomp-$LIBSECCOMP_VERSION
-RUN make -j$(nproc) -C /libseccomp-$LIBSECCOMP_VERSION check
-RUN make -C /libseccomp-$LIBSECCOMP_VERSION install
+RUN make -j$(nproc)
+RUN make -j$(nproc) check-build
+RUN cd tests && ./regression -j $(nproc)
+RUN make install
 
 ARG VERSION
 RUN mkdir -p $GOPATH/src/github.com/opencontainers/runc
 RUN git -c advice.detachedHead=false clone -b v$VERSION --depth=1 https://github.com/opencontainers/runc.git $GOPATH/src/github.com/opencontainers/runc
 WORKDIR /go/src/github.com/opencontainers/runc
 
 ARG TARGET_OS \
-  BUILD_GO_TAGS \
-  BUILD_GO_CGO_ENABLED \
-  BUILD_GO_FLAGS \
-  BUILD_GO_LDFLAGS \
-  BUILD_GO_LDFLAGS_EXTRA
+	BUILD_GO_TAGS \
+	BUILD_GO_CGO_ENABLED \
+	BUILD_GO_FLAGS \
+	BUILD_GO_LDFLAGS \
+	BUILD_GO_LDFLAGS_EXTRA
```

Comment:
Are those unrelated whitespace changes on purpose? Just asking... those things tend to create merge conflicts later on, hence I'd not do them unintentionally.

Reply:
I have some autoformatter in neovim that will autoformat docker files for me. I suppose I should figure out how to disable it.